### PR TITLE
LPS-112195 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

File: modules/sdk/project-templates/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -10,7 +10,17 @@
 	<artifactId>${artifactId}</artifactId>
 	<version>${version}</version>
 	<packaging>war</packaging>
-	<properties>
+
+		<properties>
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/pom.xml]: Need to fix this SF

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
Hi @gamerson, can you review this? If it passes your review, i want to tack on the form field templates as well. So I will send another pull

**Comment by gamerson** [comment]
@lawrence-lee  LGTM! 👍

**Comment by lawrence-lee** [comment]
Thanks!

**Comment by lawrence-lee** [comment]
new branch here: https://github.com/lawrence-lee/liferay-portal/pull/153

**Pull request closed without merging**